next/frontend/hooks/useFormExportImport.tsx

```diff
@@ -85,8 +85,7 @@ export const useGetContext = () => {
   })
 
   const { mutate: migrateFormMutate, isPending: migrateFormIsPending } = useMutation({
-    mutationFn: () =>
-      formsClient.nasesControllerMigrateForm(formId, { accessToken: 'onlyAuthenticated' }),
+    mutationFn: () => formsClient.nasesControllerMigrateForm(formId, { authStrategy: 'authOnly' }),
```

Comment:
onlyAuthenticated -> authOnly on purpose, guest users are not able (and weren't before) to run migrate form